tests/features/io_uring.t

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+. $(dirname $0)/../include.rc
+. $(dirname $0)/../volume.rc
+
+cleanup;
+
+TEST glusterd
+TEST pidof glusterd
+TEST $CLI volume create $V0 ${H0}:${B0}/${V0}-0
+TEST $CLI volume set $V0 performance.write-behind off
+TEST $CLI volume set $V0 performance.read-ahead off
+
+# Toggling works.
+TEST $CLI volume set $V0 storage.linux-io_uring on
+EXPECT "on" volume_option $V0 storage.linux-io_uring
+TEST $CLI volume set $V0 storage.linux-io_uring off
+EXPECT "off" volume_option $V0 storage.linux-io_uring
+
+# Toggling fails when volume is running.
+TEST $CLI volume start $V0
+EXPECT 'Started' volinfo_field $V0 'Status'
+TEST ! $CLI volume set $V0 storage.linux-io_uring on
+TEST $CLI volume stop $V0
+TEST  $CLI volume set $V0 storage.linux-io_uring on
+TEST $CLI volume start $V0
+EXPECT 'Started' volinfo_field $V0 'Status'
+
+#Mount and perform I/O, check for FOPS.
+TEST $GFS --volfile-id=$V0 --volfile-server=$H0 $M0;
+TEST $CLI volume profile $V0 start
+
+# Coverage for posix_io_uring_writev() and posix_io_uring_fsync().
+TEST dd if=/dev/zero of=$M0/file count=2 bs=128k oflag=sync
```

Comment:
We should have a global function that tests, aligned, unaligned, small and large blocks read and writes, with and without direct IO perhaps. (we are also not performing any data validation).

Reply:
I only tested code coverage with just dd and cat. I will add these tests too.